clustered/client/src/main/java/org/ehcache/clustered/client/internal/SimpleClusterTierManagerClientEntity.java

```diff
@@ -35,73 +32,56 @@
 import org.terracotta.exception.EntityException;
 
 import java.util.Set;
-import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
 /**
  * The client-side {@link Entity} through which clustered cache operations are performed.
  * An instance of this class is created by the {@link ClusterTierManagerClientEntityService}.
  * The server-side partner is the {@code EhcacheActiveEntity}.
  */
-public class SimpleClusterTierManagerClientEntity implements InternalClusterTierManagerClientEntity {
-
-  private static final Logger LOGGER = LoggerFactory.getLogger(SimpleClusterTierManagerClientEntity.class);
+public class SimpleClusterTierManagerClientEntity implements ClusterTierManagerClientEntity {
 
   private final EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint;
   private final LifeCycleMessageFactory messageFactory;
 
-  private Timeouts timeouts = Timeouts.builder().build();
-
   public SimpleClusterTierManagerClientEntity(EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint) {
     this.endpoint = endpoint;
     this.messageFactory = new LifeCycleMessageFactory();
   }
 
-  @Override
-  public void setTimeouts(Timeouts timeouts) {
-    this.timeouts = timeouts;
-  }
-
   @Override
   public void close() {
     endpoint.close();
   }
 
   @Override
   public void validate(ServerSideConfiguration config) throws ClusterException, TimeoutException {
-    invokeInternal(timeouts.getLifecycleOperationTimeout(), messageFactory.validateStoreManager(config), false);
+    invokeInternal(messageFactory.validateStoreManager(config), false);
   }
 
   @Override
   public Set<String> prepareForDestroy() {
     try {
-      PrepareForDestroy response = (PrepareForDestroy) invokeInternal(timeouts.getLifecycleOperationTimeout(), messageFactory
-        .prepareForDestroy(), true);
+      PrepareForDestroy response = (PrepareForDestroy) invokeInternal(messageFactory.prepareForDestroy(), true);
```

Comment:
This shouldn't timeout now right? so the TimeoutException shouldn't need to be caught here.

Reply:
Yes